extensions/wikia/Chat2/ChatBanListSpecial_helper.php

```diff
@@ -221,15 +220,14 @@ private function getUserLinks( $user ) {
 
 		if ( $this->wg->User->isAllowed( 'block' ) && ( !$userIsBlocked ) ) {
 			$links[] = Linker::link(
-				Title::newFromText( "BlockIP/{$user->getName()}", NS_SPECIAL ),
+				SpecialPage::getSafeTitleFor( 'Block', $username ),
 				$this->wg->Lang->ucfirst( wfMsg( 'blocklink' ) )
 			);
 		}
 		if ( $this->wg->User->isAllowed( 'userrights' ) && ( !$userIsBlocked ) ) {
 			$links[] = Linker::link(
-				Title::newFromText( 'UserRights', NS_SPECIAL ),
-				$this->wg->Lang->ucfirst( wfMsg( 'listgrouprights-rights' ) ),
-				"user={$oEncUserName}"
+				SpecialPage::getSafeTitleFor( 'UserRights', $username ),
+				$this->wg->Lang->ucfirst( wfMsg( 'listgrouprights-rights' ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)